test/datagateway_api/db/endpoints/test_create_db.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+
+from datagateway_api.src.common.config import Config
+
+
+def prepare_db_data_for_assertion(response_json):
+    response_json.pop("createId")
+    response_json.pop("createTime")
+    response_json.pop("id")
+    response_json.pop("modId")
+    response_json.pop("modTime")
+    response_json.pop("visitId")
+
+    if response_json["releaseDate"] is None:
+        return response_json
+
+    response_json["endDate"] = response_json["endDate"][:-6]
+    response_json["releaseDate"] = response_json["releaseDate"][:-6]
+    response_json["startDate"] = response_json["startDate"][:-6]
```

Comment:
There are entities that have a start and end date but no release date (e.g. `FacilityCycle`). I know we don't make use of any of these entities right now, but would it be better to do something similar to my suggestion? This might avoid bugs in the future if the entities in the test data is expanded. I know this is unlikely, but it _could_ save some time for a developer should this happen.

```suggestion
    response_json["endDate"] = response_json["endDate"][:-6]
    response_json["startDate"] = response_json["startDate"][:-6]
    
    if response_json["releaseDate"] is None:
        return response_json
    else:
        response_json["releaseDate"] = response_json["releaseDate"][:-6]
```

Reply:
Changed in the latest commit.